content/news/XXXX-XX-XX-Midsummer-Mixxx.md

```diff
@@ -0,0 +1,65 @@
+
+#### Let the Mixxx in
+
+A lot of new contributors found their way to our mapping parties on the discourse forum, our coding raves at github, 
+and the development zulip dancehall. You are all welcome and we'd like to thank you for all kind of input.
+With your feedback on the 2.5 beta version the coding team is on track for gold, the bugs and errors teams are already defeated.
+
+#### I'm going on a holiday and …
+
+While you're enjoying a well deserved vacation the Mixxx team is preparing new features.
+Not all new features are already for bundling in an official release but you can find everything about them in the partie venues. 
+
+The new visual aids for key and bpm will be astonishing, it needed quite some adaptations but the project is in pole position for gold.
+
+The color integration will give you intuitive directions of matching tracks for harmonic mixing, also for people who suffer off color-blindness.
+Soon you wil be able to discover a 'heat column' in the library which will give you hints on tracks that are compatible with the current playing track,
+based on the Key and BPM. And BPM coloring will be added to show tracks with a BPM close to the current tempo.
+This will be our first step in Intelligent Assistance, it can inspire you to make surprising Mixxx-es with tracks that slipped your mind.
+And these features will later be used later for some QOL features, like an improved suggestions algorithm, search ranking based on similarity,
+and adding key color ticks on the rate sliders.
+Fill the floors!
+
+Our marathon stem project is fantastic, a peek behind the curtain:
+Playing with stems changes the whole idea of DJ-ing. If stems are new for you I would like to invite you to do some research. 
+In brief: stems are regular music files that are separated in 4 stem-tracks by the original producer or with the aid of AI.
+Mostly those tracks are drums / bass / other (synths, guitar) and vocals. Together with the original track are those 4 new tracks
+combined in a new container, a *.Stem.MP4 or *.Stem.M4A file, lossy or lossless (depends also on source).
+While other DJ software will only support lossy stem files, we choose to support both.
+These combined files can be 'remixed' on the fly, with other words: the volume of the tracks can be individual controlled,
+a track can also be muted e.g. if you mute all tracks except the vocals you are actually playing an a capella.
+
+Without stems you need the beats to be synced, even if the intermediate beat would fit better to cover bass/drum sounds,
+you need to work hard with the eq's to camouflage bass left-overs. With stems you can just completely hide the unwanted tracks.
+
+Remark: The quality of the stems is of course very important, a lot of stem creators are showing up everywhere. 
+Some need you to upload your original file, some can use your GPU, some use well trained AI models, others don't.
+On Zulip I wrote a report about my quest for stems.
```

Comment:
Maybe a more general question, but do we want to encourage Mixxx user to join Zulip, or shall we reserve the space for contributor? My choice of words is probably very poor, but what I mean by that is, what should be the central place for the community? Is is the [Discourse group](https://mixxx.discourse.group/) or [Zulip](https://mixxx.zulipchat.com/#recent)?
If Zulip's target is every user of Mixxx, then I would suggest adding a link to the thread directly! Many people still don't know what Zulip is, that we have one for Mixxx and how to use it (even got some people messaging me directly there, asking me what was that all about and how to use it). I know Zulip also requires authentication to view threads, unlike Discourse.

Note that's I'm also asking because I've linked Zulip in my WIP [STEM announcement](https://github.com/mixxxdj/website/pull/359) and would like to confirm this appropriated! :) 

```suggestion
On [Zulip](https://mixxx.zulipchat.com/#narrow/stream/218389-djing-techniques/topic/Creating.20stems.2C.20what.20I.20learned.20searching.20a.20good.20tool) I wrote a report about my quest for stems.
```

Reply:
Didn't every contributor start as a user? How can a possible contributer inform him/her/themself, we write everywhere that more info can be found on Wiki & Zulip, mapping-questions can be posed on discourse en bugs/features on github.
I didn't link mu post on Zulip, I encourage people to discover it.

Reply:
We always had the forum for user related topics and for sharing mappings. The benefit is the read only access an the indexing by Search Engines (is that still the case?) 

Zulip is the successor of our developers mailing list. 

That`s not carved in stone though. At least for support questions the forum is the first address.